packages/mui-material/src/Tooltip/Tooltip.js

```diff
@@ -872,7 +872,7 @@ Tooltip.propTypes /* remove-proptypes */ = {
     PropTypes.object,
   ]),
   /**
-   * Tooltip title. Zero-length titles string are never displayed.
+   * Tooltip title. Zero-length titles string, undefined, null and false are never displayed.
    */
   title: PropTypes /* @typescript-to-proptypes-ignore */.node.isRequired,
```

Comment:
@abhinav-22-tech it because the props type does not change yet. You need to allows title to be not required as well. cc. @siriwatknp 

```suggestion
  title: PropTypes /* @typescript-to-proptypes-ignore */.node,
```